apps/portals/src/configurations/elportal/synapseConfigs/computational_tools.ts

```diff
@@ -21,9 +21,9 @@ export const computationalCardConfiguration: CardConfiguration = {
   labelLinkConfig: [
     {
       isMarkdown: false,
-      baseURL: 'Explore/Projects/DetailsPage',
+      baseURL: 'Explore/Projects',
       URLColumnName: 'grant',
-      matchColumnName: 'grant',
+      matchColumnName: 'grants',
```

Comment:
In the future it is possible for one grant to have many projects, so link to the general explore page and filter to show the projects that match the chosen grant.